src/styles/sass/App.css

```diff
@@ -0,0 +1,163 @@
+.box {
+  width: 130px;
+  height: 130px;
+  position: relative;
+  cursor: pointer;
+}
+.box .tile-cover {
+  width: 100%;
+  height: 100%;
+  display: block;
+  position: absolute;
+  border-radius: 10px;
+}
+.box .tile-cover img {
+  -o-object-fit: cover;
+     object-fit: cover;
+  border-radius: 10px;
+}
+.box .active .tile-cover {
+  transform: perspective(1000px) rotateY(180deg);
+}
+.box .active .tile-cover img {
+  display: block;
+  box-shadow: 2px 2px 10px white, -2px -2px 10px white;
+}
+.box .inactive {
+  width: 100%;
+  height: 100%;
+  display: block;
+  background: linear-gradient(200deg, navy, rgba(255, 0, 0, 0)), linear-gradient(200deg, rgb(255, 0, 234), rgba(0, 255, 0, 0)), linear-gradient(300deg, pink, rgba(0, 0, 255, 0));
+  border-radius: 10px;
+  box-shadow: 0.5px 0.5px 2px white, -0.5px -0.5px 2px white;
+}
+.box .inactive .tile-cover img {
+  display: none;
+}
+
+.gameboard {
+  width: 75%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  margin-top: 30px;
+  margin-bottom: 0;
+}
+.gameboard .tile {
+  display: flex;
+  flex-wrap: wrap;
+}
+.gameboard .box {
+  display: flex;
+  margin: 10px;
+}
+
+.full-page {
+  background-image: url(/assets/background2.jpg);
+  background-position: center;
+  background-size: cover;
+  font-size: 125%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.content {
+  flex-direction: column;
+  align-items: space-between;
+  text-align: center;
+}
+
+h1 {
+  color: white;
+  font-size: 3.1rem;
+  margin: 15px;
+  padding-top: 10px;
+  text-shadow: 1px 1px 4px rgb(255, 0, 234), -1px -1px 4px rgb(255, 0, 234);
+}
+
+h2 {
+  text-align: center;
+  width: 70%;
+  font-style: italic;
+  color: #ff8add;
+  margin: 0 auto;
+  font-size: 1.7rem;
+  text-shadow: 2px 2px 4px navy, -2px -2px 4px navy;
+}
+
+img {
+  width: 100%;
+  height: 100%;
+}
+
+.wrapper {
+  max-width: 1280px;
+  width: 90%;
+  margin: 0 auto;
+}
+
+footer {
+  background-color: navy;
+  margin-bottom: 0;
+  padding: 10px;
+}
+footer a {
+  text-align: center;
+  margin: 0;
+  font-size: 1.2rem;
+  text-decoration: none;
+  color: #ff8add;
+  text-shadow: 0.1px 0.1px white;
+}
+
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
I think these are the default React styling (could be completely wrong) but if it is, could also delete these base styling from the CSS file.